model/clusters_mgmt/v1/machine_pool_type.model

```diff
@@ -33,6 +33,9 @@ class MachinePool {
 	// The subnets upon which the nodes are created.
 	Subnets []String
 
+	// Optional keys and values that the machine pool provisioner will add as tags to all AWS resources it creates.
+	Tags [String]String
```

Comment:
Another question: Is the design of a `MachinePool` in CS to be intended to be agnostic of cloud provider? If so, adding here at this level would be a bit problematic because we are going to put AWS MachinePool specific attributes for the generic MachinePool concept

Reply:
This is right, the tags are included within the aws machine pool spec internally
This should be within [this](https://github.com/openshift-online/ocm-api-model/blob/ac7f60aa7d99e43ba2ca33d33a0af5c36ea4fc23/model/clusters_mgmt/v1/aws_machine_pool_type.model#L18) file indeed

Reply:
:+1:

Reply:
thanks! just changed this